src/test/java/utilities/Util.java

```diff
@@ -15,4 +22,14 @@ public static void wait(int seconds) {
 		}
 	}
 
-}
+	/*
+	 * Switch to Alert then confirm
+	 *
+	 * @Author: Olvia Wang
+	 */
+	public static void alertToconfirm(WebDriver driver) {
```

Comment:
1, confirmAlert is a better name
2, What if there's no alert that's present?
3, Formatting